.werft/wipe-devstaging.ts

```diff
@@ -1,23 +1,73 @@
-import { werft } from './util/shell';
+import { werft, exec } from './util/shell';
 import { wipePreviewEnvironment, listAllPreviewNamespaces } from './util/kubectl';
+import  * as fs from 'fs';
+import { sleep } from './util/util';
 
 
-async function wipeDevstaging() {
+async function wipeDevstaging(pathToKubeConfig: string) {
     const namespace_raw = process.env.NAMESPACE;
     const namespaces: string[] = [];
     if (namespace_raw === "<no value>" || !namespace_raw) {
         werft.log('wipe', "Going to wipe all namespaces");
-        listAllPreviewNamespaces("")
+        listAllPreviewNamespaces(pathToKubeConfig)
             .map(ns => namespaces.push(ns));
     } else {
         werft.log('wipe', `Going to wipe namespace ${namespace_raw}`);
         namespaces.push(namespace_raw);
     }
 
     for (const namespace of namespaces) {
-        await wipePreviewEnvironment("", "gitpod", namespace, { slice: 'wipe' });
+        await wipePreviewEnvironment(pathToKubeConfig, "gitpod", namespace, { slice: 'wipe' });
     }
-    werft.done('wipe');
 }
 
-wipeDevstaging()
\ No newline at end of file
+async function deleteExternalIp(k3sWsProxyIP: string, namespace: string) {
```

Comment:
Note: In general it would be good to find a common place for code like this, like a `k3s-cluster.ts` or `deployment.ts` or so. `wipe-staging.ts` should be just calling that stuff.

Reply:
makes sense. Let me move this to another file.